webpack/webpack.config.js

```diff
@@ -46,6 +46,7 @@ module.exports = (env, argv) => ({
       inlineSource: '.(js)$',
       chunks: ['ui'],
       inject: 'body',
+      cache: 'false'
```

Comment:
I added this line to the webpack config file, but I still need to quit and restart watch to check the changes I made to the ui.tsx file.